docs/src/nonadiabaticmodels/ldfa.md

```diff
@@ -1,19 +1,19 @@
 # Local density friction approximation (LDFA)
 
-During dynamics of molecules interacting with metals, low energy electron-hole pair
-excitations can lead to a breakdown of the Born-Oppenheimer (BO) approximation.
+During dynamics of molecules interacting with metals, interaction of nuclear motion with
+electron-hole pairs can occur and leading to a breakdown of the Born-Oppenheimer (BO) approximation.
 Molecular dynamics with electronic friction (MDEF) has been proposed as a correction to
-BO dynamics, where a Langevin equation is used to model the dynamics of the molecule
-within the metal electron density.
+BO dynamics for cases where low-energy electron hole pairs are excited. In MDEF, the Langevin 
+equation is used to model the dynamics of the molecule within the metal electron density.
 
 ![ldfa motion](../assets/figures/ldfa_motion.png)
 ![ldfa dissociate](../assets/figures/ldfa_dissociate.png)
 
-The LDFA provides a simple way to relate the electron density to a friction coefficient
-that can be used to propagate the dynamics.
+The local density friction approximation (LDFA) obtains the friction coefficient within the 
+Langevin equations from the local electron density.
 
 Here, LDFA is implemented using the LDA data provided by Gerrits et al. in
-[Phys. Rev. B 102, 155130](https://doi.org/10.1103/PhysRevB.102.155130).
+[Phys. Rev. B 102, 155130 (2020)](https://doi.org/10.1103/PhysRevB.102.155130).
 This data provides the friction coefficient evaluated at a few discrete values of
```

Comment:
This reference should be added to the references.bib file and cited using "[bib-key](@cite)" in line with the other references